project-koku/koku
# One approach to solve null values for subscription_name

## Jira Ticket

[COST-4313](https://issues.redhat.com/browse/COST-4313)

## Description

This change will replace the null values with `subscription_guid` value.

## Testing

1. Checkout Branch
2. Restart Koku
3. Hit endpoint 
`http://localhost:8000/api/cost-management/v1/reports/azure/costs/?group_by[subscription_guid]=*&filter[limit]=100`
4. Check if there are null values on some `subscription_name` keys
